deps/zlib/zlib.gyp

```diff
@@ -112,7 +112,7 @@
             }, {
               'sources': [ 'simd_stub.c', ],
             }],
-            ['arm_fpu=="neon"', {
+            ['arm_version==7', {
```

Comment:
I was referring specifically to this line RE: armv8 and checking of `arm_version`. This conditional would also need to check for armv8, otherwise optimizations will not be enabled for that arch.

Reply:
Fixed.